javascript-data-model/order-history.js

```diff
@@ -0,0 +1,15 @@
+var orderHistory = [
+  {
+    orderPlaced: ['August 4, 2020', 'July 19, 2020', 'July 4, 2020', 'July 3, 2020'],
+    total: [34, 44.53, 17.22, 138.93],
+    ShipTo: 'JS Masher',
+    orderNumber: ['114-3941689-8772232', '113-9984268-1280257', '114-2875557-9059409', '113-2883177-2648248'],
+    delivered: ['Delivered Aug 8, 2020', 'Delivered July 20, 2020', 'Delivered Jul 7, 2020', 'Delivered Jul 5, 2020'],
+    title: ['JavaScript for impatient programmers', 'The Timeless Way of Building', 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for WII U, PC. Support Turbo and Vibration Features. No Driver and No Lag-Gamecube Adapter', 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)', 'The Art of Sql'],
+    author: ['Dr.Alex Rauschmayer', 'Christopher Alexander', false, false, 'Stephane Faroult'],
+    returnDate: ['Return window closed on Sep 7, 2020', 'Return window closed on Aug 19, 2020', 'Return window closed on Aug 5, 2020', 'Return window closed on Aug 4, 2020', 'Return window closed on Aug 4, 2020'],
+    price: [31.55, 41.33, 15.98, 94.95, 33.99]
+
+  }
+];
+console.log(orderHistory);
```

Comment:
We need to re-work on this data model because there is no way to tell which products are fall under which order (for instance, there is one product per order for first 3 orders but the last order has two products)

Reply:
Do we need to place them separate(each order)?

Reply:
It's totally up to you unless it makes sense and data can be connected to each others